webgpu.h

```diff
@@ -1508,6 +1517,15 @@ typedef struct WGPUUncapturedErrorCallbackInfo {
     /*.userdata2=*/NULL _wgpu_COMMA \
 })
 
+typedef struct WGPUPostTaskHookInfo {
+    WGPUPostTaskHook hook;
```

Comment:
Have I missed it or this `WGPUPostTaskHook` symbol is never defined?

Reply:
It's hard to see because it's a function pointer type... and out of order because I wasn't thinking that hard about it:

`typedef (*WGPUPostTaskHook)(...);`